.github/workflows/release.yaml

```diff
@@ -143,6 +143,43 @@ on:
           passed as same-named input to release-notes action. Controls whether copies of
           release-notes of sub-components are included.
 
+      categories:
+        type: string
+        default: ''
+        description: |
+          comma separated list of release-note categories to include in slack posting.
+          defaults to include *all* categories.
+
+          for all known categories, see:
+          https://github.com/gardener/cc-utils/blob/master/release_notes/model.py
+
+          example:
+          value: "breaking,feature"
+
+      audiences:
```

Comment:
naming + not only relevant for posting to slack (see above)

Reply:
> not only relevant for posting to slack

We discussed that release-notes filter should only affect the slack posting, not the release-notes published via OCM. Thus the inpus must only affect the action(s) posting to slack.

What am I missing here?